lib/drivers/onetouch/catalina-sudo/sudo-askpass.osascript.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env osascript -l JavaScript
+
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2020, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/* eslint-disable */
+
+ObjC.import('stdlib')
+
+const app = Application.currentApplication()
+app.includeStandardAdditions = true
+
+const result = app.displayDialog('Please enter your computer password to allow Tidepool Uploader to access your device.', {
+  defaultAnswer: '',
+  withIcon: 'note',
```

Comment:
This doesn't appear to be a valid icon. ie, it will just use the default app icon.

Reply:
According to the [official documentation](https://developer.apple.com/library/archive/documentation/LanguagesUtilities/Conceptual/MacAutomationScriptingGuide/DisplayDialogsandAlerts.html) there _should_ be `note`, `stop` and `caution` system icons. I just tested it again with all three, and the `note` icon indeed does just display the default app icon. If I don't add this line, it doesn't display an icon at all, so I'm going to leave it as is.